hypha/static_src/src/sass/apply/abstracts/_variables.scss

```diff
@@ -64,8 +64,9 @@ $color--correct: $color--mint;
 $color--button-disabled: $color--lighter-blue;
 
 // Fonts
-$font--primary: 'noto-sans';
-$font--headings: 'montserrat';
+$font--primary: -apple-system,BlinkMacSystemFont,Segoe UI,system-ui,Roboto,Helvetica Neue,Arial,
```

Comment:
Should there not be quotes around values with spaces?

Reply:
GitHub have this: `-apple-system,BlinkMacSystemFont,"Segoe UI","Noto Sans",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji"`

Reply:
Now it says both without quotes and with quotes work for font names with spaces. 

But the spec recommends putting quotes around font names with space, so I'll add them here. Nice catch! 

More at https://stackoverflow.com/a/13752149/782901